packages/selected-network-controller/src/SelectedNetworkController.ts

```diff
@@ -82,12 +86,15 @@ export type SelectedNetworkControllerActions =
 export type AllowedActions =
   | NetworkControllerGetNetworkClientByIdAction
   | NetworkControllerGetStateAction
-  | PermissionControllerHasPermissions;
+  | PermissionControllerHasPermissions
```

Comment:
This is currently defined in this file on line 72/76, it can now be imported instead

Reply:
bump on this comment